### PR TITLE
chore: emit warning when no instance specific gated training env var is available, and raise exception when accept_eula flag is not supplied

### PR DESCRIPTION
**Pull request by evakravi**
*Issue #, if available:*

*Description of changes:*
If a user does 
```
estimator = JumpStartEstimator(model_id="huggingface-llm-gemma-7b-instruct", instance_count=1, instance_type="ml.p3.2xlarge")
```
The `estimator` object will use a dummy artifact, since there's no gated artifact available for that instance type. Instead, the desired behavior is for an exception to be raised, guiding the user to the correct instance types to choose.

Also, if a user attempts to use a gated model and does not supply `accept_eula=True` as an environment variable, raise an exception.  

*Testing done:*
Unit tests added. 

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [ ] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ ] I certify that the changes I am introducing will be backward compatible, and I have discussed concerns about this, if any, with the Python SDK team
- [ ] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [ ] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added unit and/or integration tests as appropriate to ensure backward compatibility of the changes
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [ ] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-unit-tests
* Commit ID: c554a9f689feae30caaa2eb530f8c00ad7e5ba38
* Result: FAILED
* [Build Logs](https://fegcnxu6da.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=0d268076-501b-4f5e-a6ed-f3c8802cc587%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-notebook-tests
* Commit ID: c554a9f689feae30caaa2eb530f8c00ad7e5ba38
* Result: SUCCEEDED
* [Build Logs](https://bkqkixfb59.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=ddc8c314-632b-4c7c-9d23-6bf17a1ee0d7%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: c554a9f689feae30caaa2eb530f8c00ad7e5ba38
* Result: SUCCEEDED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=79ead5b0-eeb1-40ac-9021-fbd39e5234bf%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: c554a9f689feae30caaa2eb530f8c00ad7e5ba38
* Result: FAILED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=30184448-6d88-4b31-91f6-cef923e4255b%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-slow-tests
* Commit ID: c554a9f689feae30caaa2eb530f8c00ad7e5ba38
* Result: FAILED
* [Build Logs](https://omfrydptb9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=0f45f347-91f8-4374-860d-a3bf1f47eb9d%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-unit-tests
* Commit ID: 4656cb8804004350e78e5dbed5f2c17e497c6151
* Result: FAILED
* [Build Logs](https://fegcnxu6da.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=24c75747-cea7-4354-81d1-d83a3845bbae%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: src/sagemaker/jumpstart/factory/estimator.py [hunk 1]
```diff
@@ -595,6 +596,25 @@ def _add_env_to_kwargs(
...
+        model_specs = verify_model_region_and_return_specs(
+            model_id=kwargs.model_id,
+            version=kwargs.model_version,
+            region=kwargs.region,
+            scope=JumpStartScriptScope.TRAINING,
+            tolerate_deprecated_model=kwargs.tolerate_deprecated_model,
+            tolerate_vulnerable_model=kwargs.tolerate_vulnerable_model,
+            sagemaker_session=kwargs.sagemaker_session,
+        )
+        if model_specs.is_gated_model():
```
Comment (whittech1) [inline src/sagemaker/jumpstart/factory/estimator.py:613]: Question: shouldn't this be the outer `if` statement here? And we would throw an error in all of the cases (the `environment` does not contain the special key, `accept_eula` is missing, `accept_eula` is false, etc.)?

**Review by whittech1 — APPROVED** [review]

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: 4656cb8804004350e78e5dbed5f2c17e497c6151
* Result: SUCCEEDED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=6757fdc6-dacd-4176-adcd-11aa6dc1d68e%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-notebook-tests
* Commit ID: 4656cb8804004350e78e5dbed5f2c17e497c6151
* Result: SUCCEEDED
* [Build Logs](https://bkqkixfb59.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=b5796952-6e76-4d1b-afe1-d9f7aeef70f7%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-slow-tests
* Commit ID: 4656cb8804004350e78e5dbed5f2c17e497c6151
* Result: FAILED
* [Build Logs](https://omfrydptb9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=99946e79-f53b-4cf8-9bf1-48428e002600%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-notebook-tests
* Commit ID: 03b2c9ce9c2d9d8200958da5ced55156a447c2cd
* Result: SUCCEEDED
* [Build Logs](https://bkqkixfb59.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=201a2fc5-7d30-44b4-970b-04343151daa0%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: 03b2c9ce9c2d9d8200958da5ced55156a447c2cd
* Result: FAILED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=30c0df98-b241-4b88-aa9b-7d97a5e46015%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: 03b2c9ce9c2d9d8200958da5ced55156a447c2cd
* Result: FAILED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=393e3c7d-1159-49b1-ba39-0d7e6aff7d43%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-slow-tests
* Commit ID: 03b2c9ce9c2d9d8200958da5ced55156a447c2cd
* Result: FAILED
* [Build Logs](https://omfrydptb9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=081c69ec-df43-404c-bc74-f87cc117cb4a%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-unit-tests
* Commit ID: 03b2c9ce9c2d9d8200958da5ced55156a447c2cd
* Result: FAILED
* [Build Logs](https://fegcnxu6da.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=d362f391-961b-4182-9932-e4b733395181%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: src/sagemaker/jumpstart/factory/estimator.py (hunk 1 above)
Comment (JGuinegagne) [reply src/sagemaker/jumpstart/factory/estimator.py:613]: +1, this is odd, why bother retrieving the specs if you're not going to use them?

File: src/sagemaker/jumpstart/types.py [hunk 2]
```diff
@@ -956,6 +956,10 @@ def use_training_model_artifact(self) -> bool:
         # otherwise, return true is a training model package is not set
         return len(self.training_model_package_artifact_uris or {}) == 0
 
+    def is_gated_model(self) -> bool:
+        """Returns True if the model has a EULA key or the model bucket is gated."""
+        return self.gated_bucket or self.hosting_eula_key is not None
```
Comment (JGuinegagne) [inline src/sagemaker/jumpstart/types.py:961]: hum, this is odd, shouldn't the pySDK only us `gated_bucket` as indicator?

Isn't it MH's unit test job to ensure that if `hosting_eula_key` is passed, then `gated_bucket` is set correctly?

File: src/sagemaker/jumpstart/utils.py [hunk 3]
```diff
@@ -466,21 +466,25 @@ def update_inference_tags_with_jumpstart_training_tags(
     return inference_tags
 
 
+def get_eula_message(model_specs: JumpStartModelSpecs, region: str) -> str:
+    """Returns EULA message to display to customers if one is available, else empty string."""
```
Comment (JGuinegagne) [inline src/sagemaker/jumpstart/utils.py:470]: remove 'to customers' from docstring.

**Review by JGuinegagne — CHANGES_REQUESTED** [review]

File: src/sagemaker/jumpstart/types.py (hunk 2 above)
Comment (evakravi) [reply src/sagemaker/jumpstart/types.py:961]: We don't have a unit test for this (tho we should). It was confusing (at least for me) to refer to gated buckets and hosting eula keys separately, so to clear the confusion i created this helper function to clarify these are the markers for gated models (if the bucket is gated or there's a eula key).

File: src/sagemaker/jumpstart/factory/estimator.py (hunk 1 above)
Comment (evakravi) [reply src/sagemaker/jumpstart/factory/estimator.py:613]: fwiw, this won't involve another s3 call, it's just reading from memory

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: e2397790e39895b95b16f69adaebe40ffd2e4d2e
* Result: SUCCEEDED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=8d994547-9f36-4c7d-85da-52806c23157f%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-notebook-tests
* Commit ID: e2397790e39895b95b16f69adaebe40ffd2e4d2e
* Result: SUCCEEDED
* [Build Logs](https://bkqkixfb59.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=30c7a617-9c34-4c3c-b928-dcda3125d441%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: e2397790e39895b95b16f69adaebe40ffd2e4d2e
* Result: FAILED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=b6065520-f46f-4538-a53a-ef4b2e093076%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-unit-tests
* Commit ID: 8bc10e81e07bec5fdfb94ea5fd883fdcecf5f3a9
* Result: FAILED
* [Build Logs](https://fegcnxu6da.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=d8e9275e-3cc4-4395-84df-dc67b7a1748c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: 8bc10e81e07bec5fdfb94ea5fd883fdcecf5f3a9
* Result: FAILED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=575ce5bb-aaba-4d00-a0d2-5482800cdd16%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: 8bc10e81e07bec5fdfb94ea5fd883fdcecf5f3a9
* Result: SUCCEEDED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=f382ac51-db1b-4aed-bba1-e24410912755%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-slow-tests
* Commit ID: 8bc10e81e07bec5fdfb94ea5fd883fdcecf5f3a9
* Result: FAILED
* [Build Logs](https://omfrydptb9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=d6dbdf9d-4ee3-4395-a38f-5a58af19e281%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-notebook-tests
* Commit ID: 8fffcc9ca54f7691ee8dc12a22e0340799576159
* Result: SUCCEEDED
* [Build Logs](https://bkqkixfb59.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=4281d66f-9e82-4688-9d82-c6ffd7c6e1ee%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: 8fffcc9ca54f7691ee8dc12a22e0340799576159
* Result: FAILED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=bbc0d160-a35c-45bb-9551-bf990f7ff0ae%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: 8fffcc9ca54f7691ee8dc12a22e0340799576159
* Result: SUCCEEDED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=ffd1741c-d9ff-46cd-817c-0b9e941440d4%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: bf0bc96d4019d73b3f7186b9b059a02d481baf32
* Result: FAILED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=79832562-4dd7-4d5a-9779-80743a8e0cff%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-notebook-tests
* Commit ID: bf0bc96d4019d73b3f7186b9b059a02d481baf32
* Result: SUCCEEDED
* [Build Logs](https://bkqkixfb59.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=c8fc932e-5064-4a26-b412-b0d863abe05c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: bf0bc96d4019d73b3f7186b9b059a02d481baf32
* Result: FAILED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=14416c85-a1fc-496a-9377-ebf9b5021445%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by JGuinegagne — APPROVED** [review]

**Review by liujiaorr — COMMENTED** [review]
/Bot run pr

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: bf0bc96d4019d73b3f7186b9b059a02d481baf32
* Result: FAILED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=9acf0828-4135-4a6a-bbe9-f114c1746bff%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-notebook-tests
* Commit ID: bf0bc96d4019d73b3f7186b9b059a02d481baf32
* Result: SUCCEEDED
* [Build Logs](https://bkqkixfb59.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=7022f274-cedb-4215-84ee-d40bb8838c12%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-slow-tests
* Commit ID: bf0bc96d4019d73b3f7186b9b059a02d481baf32
* Result: FAILED
* [Build Logs](https://omfrydptb9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=4a8f1427-792e-4d44-86e6-20b66c4c36e7%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by liujiaorr — APPROVED** [review]

**Pull request merged**